cloudvcl/settings.py

```diff
@@ -119,6 +119,10 @@
 
 STATIC_URL = '/static/'
 
+# Location on the hard drive where we want files uploaded to
```

Comment:
Isn't that the default? Anyway, it's not relevant because we don't save nor serve those uploaded.
Remove